src/officehours_api/backends/bluejeans.py

```diff
@@ -127,7 +129,11 @@ def update_meeting(self, user_id, meeting_id, meeting):
 
 
 class Backend:
-    friendly_name = 'BlueJeans'
+    name: str = 'bluejeans'
+    friendly_name: str = 'BlueJeans'
+    docs_url: Union[str, None] = settings.BLUEJEANS_DOCS_URL
+    telephone_num: Union[str, None] = settings.BLUEJEANS_TELE_NUM
```

Comment:
I can change it to `Optional`. Some folks I work with prefer the `Union` formulation since it's more explicit. I don't have strong opinions either way.